packages/web/src/pages/chat-page/components/ChatComposer.module.css

```diff
@@ -7,20 +7,50 @@
   padding: var(--unit-2) var(--unit-4);
 }
 
+.input {
+  padding: var(--unit-1) var(--unit-4);
+}
+
+.input textarea {
+  /* Compensate for the larger icon */
+  line-height: 200%;
+}
+
 .icon {
-  width: 1.5em;
-  height: 1.5em;
-  display: block; /* Prevent descender space */
+  width: 20px;
+  height: 20px;
+  /* 1 pixel nudge to visually center */
+  position: relative;
+  left: -1px;
```

Comment:
This feels off... Is the icon not centered in the svg?

Reply:
I believe it's correctly centered in the SVG (and we use it in many places) – I believe it's due to the icon's shape that it being actually centered looks wrong.

Without 1px shift (looks weird)
<img width="256" alt="Screen Shot 2023-06-06 at 9 43 25 PM" src="https://github.com/AudiusProject/audius-client/assets/6780028/aa1e78cf-4e57-49b9-a194-40b08e537845">

And yet, the SVG appears centered:

<img width="240" alt="Screen Shot 2023-06-06 at 9 44 00 PM" src="https://github.com/AudiusProject/audius-client/assets/6780028/520b9cd9-c550-418a-9c29-4f1a9b6aa8f8">

<img width="360" alt="Screen Shot 2023-06-06 at 9 37 41 PM" src="https://github.com/AudiusProject/audius-client/assets/6780028/0982a0f9-d523-419c-a426-4369a7ae7cb4">